autoprotocol/instruction.py

```diff
@@ -250,7 +257,22 @@ def __init__(self, groups, magnetic_head):
                 f"container_types: {self.heads[magnetic_head]} for head_type: "
                 f"{magnetic_head}"
             )
-
+        non_valid_well_volumes = [
```

Comment:
Does this contain well volumes or well objects with with volumes above some threshold?

Reply:
`wells_with_invalid_volume` is what I think this should be